CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ cmake_minimum_required(VERSION 2.8)
 
 set(PROJECT_NAME mc-mini)
 set(PROJECT_VERSION 1.1.0)
-project(${PROJECT_NAME})
+
+if(CMAKE_VERSION LESS 3.0)
```

Comment:
I think I tried this before and it didn't pass Travis CI, but I'm not certain whether I did it. I completely agree with you here, of course. Let me try this again and we'll see if it passes.